--- samhain-1.5.4/Makefile.in	Mon Aug  5 11:09:15 2002
+++ samhain-1.5.4/Makefile.in	Mon Aug  5 11:09:48 2002
@@ -310,13 +310,13 @@
 
 
 install-program: $(PROGRAMS) sstrip
-	@$(mkinstalldirs) $(sbindir)
+	@$(mkinstalldirs) $(DESTDIR)$(sbindir)
 	@list='$(PROGRAMS)'; for p in $$list; do \
 	  if test -f $$p; then \
-	    echo " $(INSTALL_PROGRAM) $$p $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
-	    $(INSTALL_PROGRAM) $$p $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
-	    echo " ./sstrip $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
-	    ./sstrip $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
+	    echo " $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
+	    $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
+	    echo " ./sstrip $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
+	    ./sstrip $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
 	  else :; fi; \
 	done
 
@@ -335,11 +335,11 @@
 #
 
 install-data: trustfile
-	@$(mkinstalldirs) $(sysconfdir)
-	@$(mkinstalldirs) $(mylockdir)
-	@$(mkinstalldirs) $(mylogdir)
-	@$(mkinstalldirs) $(mydatadir); \
-	chmod 700 $(mydatadir)
+	@$(mkinstalldirs) $(DESTDIR)$(sysconfdir)
+	@$(mkinstalldirs) $(DESTDIR)$(mylockdir)
+	@$(mkinstalldirs) $(DESTDIR)$(mylogdir)
+	@$(mkinstalldirs) $(DESTDIR)$(mydatadir); \
+	chmod 700 $(DESTDIR)$(mydatadir)
 	@if test -f samhainrc.$(selectconfig); then \
 		:; \
 	else \
@@ -383,15 +383,15 @@
 #
 
 install-man: 
-	$(mkinstalldirs) $(mandir)/man8
-	$(mkinstalldirs) $(mandir)/man5
+	$(mkinstalldirs) $(DESTDIR)$(mandir)/man8
+	$(mkinstalldirs) $(DESTDIR)$(mandir)/man5
 	@if test -f $(srcdir)/samhain.8 ; then \
-	    echo "  $(INSTALL_DATA) $(srcdir)/samhain.8 $(mandir)/man8/@install_name@.8"; \
-	     $(INSTALL_DATA) $(srcdir)/samhain.8 $(mandir)/man8/@install_name@.8; \
+	    echo "  $(INSTALL_DATA) $(srcdir)/samhain.8 $(DESTDIR)$(mandir)/man8/@install_name@.8"; \
+	     $(INSTALL_DATA) $(srcdir)/samhain.8 $(DESTDIR)$(mandir)/man8/@install_name@.8; \
 	fi
 	@if test -f $(srcdir)/samhainrc.5 ; then \
-	    echo "  $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(mandir)/man5/@install_name@rc.5"; \
-	     $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(mandir)/man5/@install_name@rc.5; \
+	    echo "  $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(DESTDIR)$(mandir)/man5/@install_name@rc.5"; \
+	     $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(DESTDIR)$(mandir)/man5/@install_name@rc.5; \
 	fi
 
 uninstall-man:
@@ -589,9 +589,9 @@
 	rm -rf $(sbindir)/deploy.sh
 
 install-deploy: $(DEPLOYINSTALLFILES)  deploy.sh
-	$(mkinstalldirs) $(sbindir)
-	$(mkinstalldirs) $(mydatadir)/profiles/src
-	$(INSTALL_DATA) Makefile $(mydatadir)/profiles/src/Makefile
+	$(mkinstalldirs) $(DESTDIR)$(sbindir)
+	$(mkinstalldirs) $(DESTDIR)$(mydatadir)/profiles/src
+	$(INSTALL_DATA) Makefile $(DESTDIR)$(mydatadir)/profiles/src/Makefile
 	@echo "copy source files to $(mydatadir)/profiles/src"
 	@for file in $(DEPLOYINSTALLFILES); do \
 	  d=$(srcdir); \
