diff -urN samhain-1.5.4/Makefile.in samhain-1.5.4.dzikus/Makefile.in
--- samhain-1.5.4/Makefile.in	Wed Jul 24 12:51:06 2002
+++ samhain-1.5.4.dzikus/Makefile.in	Wed Jul 24 12:55:16 2002
@@ -315,8 +315,6 @@
 	  if test -f $$p; then \
 	    echo " $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
 	    $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
-	    echo " ./sstrip $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
-	    ./sstrip $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
 	  else :; fi; \
 	done
 
