diff -uNr samhain-1.3.5-old/Makefile.in samhain-1.3.5/Makefile.in
--- samhain-1.3.5-old/Makefile.in	Sun Dec 16 22:56:57 2001
+++ samhain-1.3.5/Makefile.in	Mon Jan  7 11:57:22 2002
@@ -599,22 +599,22 @@
 install-exec-am: install-binPROGRAMS install-nobinDATA
 
 install-man: 
-	$(mkinstalldirs) $(mymandir)/man8
-	$(mkinstalldirs) $(mymandir)/man5
+	$(mkinstalldirs) $(DESTDIR)$(mymandir)/man8
+	$(mkinstalldirs) $(DESTDIR)$(mymandir)/man5
 	@if test -f $(srcdir)/samhain.8 ; then \
 	    echo "  $(INSTALL_DATA) $(srcdir)/samhain.8 $(mymandir)/man8/@install_name@.8"; \
-	     $(INSTALL_DATA) $(srcdir)/samhain.8 $(mymandir)/man8/@install_name@.8; \
+	     $(INSTALL_DATA) $(srcdir)/samhain.8 $(DESTDIR)$(mymandir)/man8/@install_name@.8; \
 	fi
 	@if test -f $(srcdir)/samhainrc.5 ; then \
 	    echo "  $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(mymandir)/man5/@install_name@rc.5"; \
-	     $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(mymandir)/man5/@install_name@rc.5; \
+	     $(INSTALL_DATA) $(srcdir)/samhainrc.5 $(DESTDIR)$(mymandir)/man5/@install_name@rc.5; \
 	fi
 
 install-nobinDATA:
 	@$(NORMAL_INSTALL)
-	@$(mkinstalldirs) $(configdir)
-	@$(mkinstalldirs) $(mydatadir); \
-	chmod 700 $(mydatadir)
+	@$(mkinstalldirs) $(DESTDIR)$(configdir)
+	@$(mkinstalldirs) $(DESTDIR)$(mydatadir); \
+	chmod 700 $(DESTDIR)$(mydatadir)
 	@list='$(bin_PROGRAMS)'; for p in $$list; do \
 	  if test "x$$p" = "xsamhain_stealth"; then \
 	    echo " "; \
@@ -631,7 +631,7 @@
 	if test "x$$stealthflag" = "x"; then \
 	  if test -f $(srcdir)/samhainrc ; then \
 	    echo " $(INSTALL_DATA) $(srcdir)/samhainrc $(configfile)"; \
-	    $(INSTALL_DATA) $(srcdir)/samhainrc $(configfile); \
+	    $(INSTALL_DATA) $(srcdir)/samhainrc $(DESTDIR)$(configfile); \
 	    echo " "; \
 	    echo " Installed samhainrc to $(configfile)"; \
 	  else :; fi; \
@@ -639,8 +639,8 @@
 
 install-yuleDATA:
 	@$(NORMAL_INSTALL)
-	@$(mkinstalldirs) $(configdir)
-	@$(mkinstalldirs) $(mydatadir); \
+	@$(mkinstalldirs) $(DESTDIR)$(configdir)
+	@$(mkinstalldirs) $(DESTDIR)$(mydatadir); \
 	chmod 700 $(mydatadir)
 	@list='$(bin_PROGRAMS)'; for p in $$list; do \
 	  if test "x$$p" = "xsamhain_stealth"; then \
@@ -658,7 +658,7 @@
 	if test "x$$stealthflag" = "x"; then \
 	  if test -f $(srcdir)/yulerc.template ; then \
 	    echo " $(INSTALL_DATA) $(srcdir)/yulerc.template $(configfile)"; \
-	    $(INSTALL_DATA) $(srcdir)/yulerc.template $(configfile); \
+	    $(INSTALL_DATA) $(srcdir)/yulerc.template $(DESTDIR)$(configfile); \
 	    echo " "; \
 	    echo " Installed yulerc.template to $(configfile)"; \
 	  else :; fi; \
@@ -709,20 +709,20 @@
 
 install-binPROGRAMS: $(bin_PROGRAMS) sstrip install-lkm
 	@$(NORMAL_INSTALL)
-	@$(mkinstalldirs) $(sbindir)
+	@$(mkinstalldirs) $(DESTDIR)$(sbindir)
 	@list='$(bin_PROGRAMS)'; for p in $$list; do \
 	  if test -f $$p; then \
 	    echo " strip $$p && sstrip $$p"; \
 	    echo " $(INSTALL_PROGRAM) $$p $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
 	    strip $$p > /dev/null 2>&1; \
 	    ./sstrip  $$p; \
-	    $(INSTALL_PROGRAM) $$p $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
+	    $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
 	  else :; fi; \
 	done
 
 install-yulePROGRAMS: yule sstrip install-lkm
 	@$(NORMAL_INSTALL)
-	@$(mkinstalldirs) $(sbindir)
+	@$(mkinstalldirs) $(DESTDIR)$(sbindir)
 	@list='yule'; for p in $$list; do \
 	  if test -f $$p; then \
 	    echo " strip $$p && sstrip $$p"; \
@@ -734,9 +734,9 @@
 	    strip $$p > /dev/null 2>&1; \
 	    ./sstrip  $$p; \
 	    if test "x@install_name@" == "xsamhain"; then \
-	      $(INSTALL_PROGRAM) $$p $(sbindir)/$$p; \
+	      $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/$$p; \
 	    else \
-	      $(INSTALL_PROGRAM) $$p $(sbindir)/`echo $$p|sed 's%yule%@install_name@%'`; \
+	      $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%yule%@install_name@%'`; \
 	    fi; \
 	  else :; fi; \
 	done
@@ -784,14 +784,14 @@
 	if test "x@sh_lkm@" != "x"; then \
 		if test -d /lib/modules/$$RVER/misc; then \
 			echo " cp samhain_hide.o /lib/modules/$$RVER/misc/`echo samhain_hide.o|sed 's%samhain%@install_name@%'`"; \
-	  		cp samhain_hide.o /lib/modules/$$RVER/misc/`echo samhain_hide.o|sed 's%samhain%@install_name@%'` ; \
+	  		cp samhain_hide.o $(DESTDIR)/lib/modules/$$RVER/misc/`echo samhain_hide.o|sed 's%samhain%@install_name@%'` ; \
 			echo " cp samhain_erase.o /lib/modules/$$RVER/misc/`echo samhain_erase.o|sed 's%samhain%@install_name@%'`"; \
-	  		cp samhain_erase.o /lib/modules/$$RVER/misc/`echo samhain_erase.o|sed 's%samhain%@install_name@%'` ; \
+	  		cp samhain_erase.o $(DESTDIR)/lib/modules/$$RVER/misc/`echo samhain_erase.o|sed 's%samhain%@install_name@%'` ; \
 		elif test -d /lib/modules/misc; then \
 	  		echo " cp samhain_hide.o /lib/modules/misc/`echo samhain_hide.o|sed 's%samhain%@install_name@%'`"; \
-	  		cp samhain_hide.o /lib/modules/misc/`echo samhain_hide.o|sed 's%samhain%@install_name@%'`; \
+	  		cp samhain_hide.o $(DESTDIR)/lib/modules/misc/`echo samhain_hide.o|sed 's%samhain%@install_name@%'`; \
 			echo " cp samhain_erase.o /lib/modules/misc/`echo samhain_erase.o|sed 's%samhain%@install_name@%'`"; \
-	  		cp samhain_erase.o /lib/modules/misc/`echo samhain_erase.o|sed 's%samhain%@install_name@%'` ; \
+	  		cp samhain_erase.o $(DESTDIR)/lib/modules/misc/`echo samhain_erase.o|sed 's%samhain%@install_name@%'` ; \
 		else \
 	  		echo " install-lkm: No module directory found."; \
 		fi; \
