diff -uNr samhain-1.3.5-old/Makefile.in samhain-1.3.5/Makefile.in
--- samhain-1.3.5-old/Makefile.in	Mon Jan  7 11:57:22 2002
+++ samhain-1.3.5/Makefile.in	Mon Jan  7 12:12:57 2002
@@ -798,8 +798,6 @@
 	  if test -f $$p; then \
 	    echo " strip $$p && sstrip $$p"; \
 	    echo " $(INSTALL_PROGRAM) $$p $(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`"; \
-	    strip $$p > /dev/null 2>&1; \
-	    ./sstrip  $$p; \
 	    $(INSTALL_PROGRAM) $$p $(DESTDIR)$(sbindir)/`echo $$p|sed 's%samhain%@install_name@%'`; \
 	  else :; fi; \
 	done
